src/TableRow.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const TableRow = props => {
+  const [handleClick, setHandleClick] = useState(false);
```

Comment:
`handleClick` sounds like the name of a function and not the name of a *boolean*

Reply:
Hint: a good name for a variable would be something that starts with `is` or `has` something like that
Also think about what this state value represents